keyboards/hineybush/h87a/config.h

```diff
@@ -16,10 +16,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x0000
+#define VENDOR_ID       0x04D8
```

Comment:
You are aware that's microchips vendor ID?

Reply:
In the VIA how-to (coming soon), I am suggesting using two ASCII characters representing the "vendor", in this case, something like "HB" would make sense ;-)

Reply:
Yes, I'm aware. I did this:

https://www.microchip.com/usblicensing/

Reply:
Ah great, did not know that was an option. Thanks for clarifying